server/service/hosts.go

```diff
@@ -876,6 +878,30 @@ type hostsReportResponse struct {
 func (r hostsReportResponse) error() error { return r.Err }
 
 func (r hostsReportResponse) hijackRender(ctx context.Context, w http.ResponseWriter) {
+	// post-process the Device Mappings for CSV rendering
+	for _, h := range r.Hosts {
+		if h.DeviceMapping != nil {
+			// return the list of emails, comma-separated, as part of that single CSV field
```

Comment:
The commas may mess up the CSV?

Reply:
No this is handled by the encoding (it adds quotes around the value), and tested in the integration test.